clock/db/migrate/20201103214032_create_employees.rb

```diff
@@ -0,0 +1,18 @@
+class CreateEmployees < ActiveRecord::Migration[6.0]
```

Comment:
read this recommendation, follow the [link](https://github.com/Brightcoders-Bootcamps/kata-07-el-reloj-checador-TM/pull/9#discussion_r515331399)

Reply:
In this case we use the Rails scaffold, which is why both the CRUD generation and the migrations appear in the same code. We will consider the observation, for future reviews.